terraform/gcp/main.tf

```diff
@@ -54,9 +54,14 @@ locals {
   ) : local.zonal_permissions
 }
 
+resource "random_string" "suffix" {
+  length  = 4
+  special = false
+}
+
 resource "google_project_iam_custom_role" "autoneg" {
   project     = var.project_id
-  role_id     = var.regional ? "autonegRegional" : "autonegZonal"
+  role_id     = "autoneg${var.regional ? "Regional" : "Zonal"}.${random_string.suffix.result}"
```

Comment:
Can we add a variable like "add_random_suffix" to trigger this? And also not add the dot in case that's not turnedf on?

Reply:
Thanks @rosmo , I processed your feedback and pushed a different commit.